Changed file to to fix bug where fdp base uri could not be replaced

Shamanou opened the pull request:
- Only small changes where needed, removed return and moved getfaircalaogsmethod downwards

Shamanou committed:
Changed file to to fix bug where fdp base uri could not be replaced

- Only small changes where needed, removed return and moved getfaircalaogsmethod downwards

kburger commented on extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js:
```diff
@@ -125,11 +125,10 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
 			self._pushtoResourceDiv.html('');
 			self.uri = self._baseUriSpan.text();
-			getFairCatalogs(self._baseUriSpan.text(), self);
-			return;
```
Removing the `return` statement might have some side-effects?

Shamanou replied:
i do not foresee unexpected side effects which ones are you thinking off?

kburger commented on extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js:
```diff
@@ -125,11 +125,10 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
 		}
 		MenuSystem.dismissAll();
 		self._replaceBaseUri(newBaseUri, false);
+		getFairCatalogs(self._baseUriSpan.text(), self);
```
Could you explain a bit more about this fix? When was the if clause above called? As far as I can see, the baseuri length > 7 conditional is true even if you add a new url?

Shamanou replied:
the clause is called when the user presses the apply button within the push to fdp dialog, the user first presses the edit button to change the baseuri. The `self._replaceBaseUri` call takes care off setting the new `self.fairDataPointPost.baseUri`

kburger reviewed (commented):
Some questions, the cause is not entirely clear to me.

Shamanou commented:
The cause whas the `return;` within the if-block

Shamanou committed:
Added end user error when fdp is unavailable

End user will see an error when the FDP at the url provided cannot return metadata. Also removed duplicate line, which caused fdp to be queried twice and errors to be shown twice

kburger commented on extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js:
```diff
@@ -128,7 +128,6 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
 		}
 		MenuSystem.dismissAll();
 		self._replaceBaseUri(newBaseUri, false);
-		getFairCatalogs(self._baseUriSpan.text(), self);
```
Was this line supposed to be removed? Seems like the retrieval of the catalogs is not triggered from the apply event now.
